Shopify/cli
# Feature/support parsing structured logs

### WHY are these changes introduced?

Fixes [#364](https://github.com/Shopify/cli/pull/364)
The Extension server is now logging structured logs and the CLI needs to be able to parse them in to the desired format.
The problem is describe further in this [document](https://docs.google.com/document/d/1EaxKuiDOHuukkqayNr_Ot-7JcAwomjhDYnLcwfNuLko/edit#)


### WHAT is this pull request doing?
Added Structured logging to the Extension server and parsing of the structured logging in the CLI

### How to test your changes?

- Create an App through the CLI `yarn create app --local`
- Go the the app folder and create an extension in the app `yarn scaffold extension`
- Serve the app&extension `yarn dev`
- check the output logs
- Introduce a breaking change in the extension template like changing an HTML tag and see the error log
